setup.py

```diff
@@ -19,24 +19,63 @@
 """
 
 import setuptools
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
 
 # Get the long description from the README file
 long_description = (here / 'README.md').read_text(encoding='utf-8')
 
+
 setuptools.setup(
-    name="arc3o-ClimateClara", 
-    version="0.1",
-    author="Clara Burgard",
-    author_email="clara.burgard@gmail.com",
-    description="The Arctic Ocean Observation Operator for 6.9 GHz",
+	
+	#The project's name
+    name='arc3o',
+    
+    #The project's version 
+    version='0.1',
+    
+    #The project's metadata
+    author='Clara Burgard',
+    author_email='clara.burgard@gmail.com',
+    description='An observation operator for the Arctic Ocean for 6.9 GHz',
     long_description=long_description,
-    long_description_content_type="text/markdown",
-    url="https://github.com/ClimateClara/arc3o",
-    packages=setuptools.find_packages(),
+    
+    #The project's main homepage.
+    url='https://github.com/ClimateClara/arc3o',
+    
+    #The project's license
     license='GPL-3.0',
+    
+    packages=find_packages(exclude=['docs', 'tests*', 'examples']),
+    
     classifiers=[
-        "Programming Language :: Python :: 3",
-        "Operating System :: OS Independent",
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Climate scientists',
+        'License :: OSI Approved :: GNU General Public License v3 (GPL-3.0)',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
     ],
-    python_requires='>=3.6',
+    
+    install_requires=[
+          'numpy',
+          'xarray',
+          'pandas',
+          'datetime',
+          'time',
+          'timeit',
+          'itertools',
+          'tqdm',
+          'subprocess',
+          'pathos.multiprocessing'
+      ],
+      
+    project_urls={
+    	'Bug Reports': 'https://github.com/ClimateClara/arc3o/issues',
```

Comment:
```suggestion
        'Source': 'https://github.com/ClimateClara/arc3o',
        'Tracker': 'https://github.com/ClimateClara/arc3o/issues',
```

see https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls